Yu_Zhu_Sharma.sh

```diff
@@ -0,0 +1,53 @@
+#usage: this script could be run in the bioinformaticsproject folder, bash Yu_Zhu_Sharma.sh 
+
+#make a directory for output file, and if script needs to be run again, run rm -r output before running this script
+mkdir output
+
+#put all mcrA in one file
+for file in ~/Private/Biocomputing2022/bioinformaticsProject/ref_sequences/m*.fasta
+do 
+cat $file >> ~/Private/Biocomputing2022/bioinformaticsProject/output/mcrAall.txt
+done
+
+
+#put all hsp in one file
+for file in ~/Private/Biocomputing2022/bioinformaticsProject/ref_sequences/h*.fasta
+do 
+cat $file >> ~/Private/Biocomputing2022/bioinformaticsProject/output/hspall.txt
+done
+
```

Comment:
+2

Reply:
You can make it without the for loop
cat mcrAgene_* > ref_mcrAgenes.fasta                                                                                                                          
cat hsp70gene_* > ref_hsp70genes.fasta